### DISCUSSION TITLE
plugin joit_pwmdir and OUTPUT_SCALE in ini file.

### DISCUSSION THREAD

**Discussion by zayoo-HR** [multigcs/LinuxCNC-RIO#42] [General]
Hello,

please can somebody explain me how to calculate this factor if i drive DC motor in closed loop with pwmdir plugin. 

If my motor give me 19.68mm/s at 24V i get OUTPUT_SCALE = 0.82[mm/secvolt) with this factor no way to start system. With default 400 work surprisingly well.

**Comment by multigcs** [comment]
sorry, that's probably my mistake,
i have no experience with servo motors,
so the scaling is probably not implemented correctly ;(
